eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -100,6 +100,13 @@ stages:
                           packagesToPush: '$(Pipeline.Workspace)/staging/**/*.nupkg;!$(Pipeline.Workspace)/staging/**/*.symbols.nupkg'
                           nuGetFeedType: external
                           publishFeedCredentials: Nuget.org
+                      - task: NuGetCommand@2
+                        condition: ne(variables['DevOpsFeedID'], '')
```

Comment:
Given that you have a condition here you will need to explicitly add the succeeded condition to it as well otherwise it will try to run even if the previous steps fail. `and(succeeded(),  ne(variables['DevOpsFeedID'], ''))`